site/app/models/OfficeHoursQueueModel.php

```diff
@@ -62,6 +88,10 @@ public function isGrader() {
         return $this->core->getUser()->accessGrading();
     }
 
+    public function accessFullGrading() {
```

Comment:
This shouldn't be a function here, we typically want to separate the model and view and avoid passing `$core` unless we have too. the value of `$this->core->getUser()->accessFullGrading()` should be passed from the model/controller to the view instead.